### ISSUE TITLE
Branch 2.4.x crashes on startup

### ISSUE THREAD

**Issue by stephen-workpop** [BranchMetrics/android-branch-deep-linking#304]
I had the following line in my Gradle file:
`  
compile 'io.branch.sdk.android:library:2.+'
`

Today, my app started crashing on startup with the following stack:
```
E/AndroidRuntime: FATAL EXCEPTION: main
Process: com.workpop.emp.debug, PID: 26876
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.support.customtabs.CustomTabsSession.mayLaunchUrl(android.net.Uri, android.os.Bundle, java.util.List)' on a null object reference
at io.branch.referral.BranchStrongMatchHelper$2.onCustomTabsServiceConnected(BranchStrongMatchHelper.java:64)
at android.support.customtabs.CustomTabsServiceConnection.onServiceConnected(CustomTabsServiceConnection.java:32)
at android.app.LoadedApk$ServiceDispatcher.doConnected(LoadedApk.java:1223)
at android.app.LoadedApk$ServiceDispatcher$RunConnection.run(LoadedApk.java:1240)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

I figured out that if I downgraded past 2.4 to 2.3, the crash no longer happens. Now my Gradle file has the following line:
`
compile 'io.branch.sdk.android:library:2.3.1'
`

Any idea why my app might be crashing on later versions?

**Comment by sojanpr** [comment]
@stephen-workpop Thanks for notifying this. It seems like some times the chrome tab returns `null` for sessions. We have added a null check in the pr https://github.com/BranchMetrics/android-branch-deep-linking/pull/305
 We have release this fix with release 2.4.4 . It is available on Maven. Could you please try it and update the  issue

**Comment by stephen-workpop** [comment]
No more crashes with 2.4.4 thanks!
